interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -3789,16 +3789,75 @@ function document_engine($pid)
         if (($row1['value'] ?? '') > '') {
             //if there is a value, tells us what segment of exam ($zone) this belongs in...
             $zones[$row1['value']][] = $row1;
+            $zone[$row1['name']] = true;
         } else {
             if ($row1['name'] != "Categories") {
                 $zones['OTHER'][] = $row1;
             }
         }
     }
 
+    if (!$zone['AntSeg Laser']) {
+        $sql = "select id from categories ORDER by id desc LIMIT 1";
+        $last_row = sqlQuery($sql);
+        $counter = $last_row['id'];
+        $counter++;
+        $sql = "INSERT INTO `categories` ( `id`, `name`, `value`, `parent`, `aco_spec`, `codes`)
+                    VALUES (?, 'AntSeg Laser', 'ANTSEG', '14', 'patients|docs', '');";
```

Comment:
this doesn't look right. can't assume parent is 14 and lft rht columns are left blank (would check out the upgrade scripts for adding categories (it is a bit complicated but can just copy algorithm in the upgrade scripts))